express-hello-world/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res, next) {
+  res.send('Hello World');
+  next();
+});
+
+app.listen(3000, () => {
+});
```

Comment:
It's generally a good idea to print out a message when the server is ready to accept requests. Production servers can take several minutes to start.

Reply:
Should I just do the eslint comment thingy to allow me to console.log?